library/data/component/src/main/java/org/quiltmc/qsl/component/impl/ComponentsImpl.java

```diff
@@ -38,60 +38,65 @@
 
 @ApiStatus.Internal
 public class ComponentsImpl {
-	private static final Logger LOGGER = LoggerFactory.getLogger("QSL/Components");
-	public static final String MODID = "quilt_component";
-    private static final Map<ComponentInjectionPredicate, Set<Identifier>> INJECTION_REGISTRY = new HashMap<>();
-	private static final RegistryKey<Registry<ComponentType<?>>> REGISTRY_KEY =
-			RegistryKey.ofRegistry(new Identifier("quilt", "components"));
+	public static final RegistryKey<Registry<ComponentType<?>>> REGISTRY_KEY =
```

Comment:
When I said "exposed", I meant in the API (in `Components`, preferably)! Sorry for the confusion.

Reply:
Oh, very well then!

Reply:
![image](https://user-images.githubusercontent.com/46004440/176508281-c817f4a3-c464-40df-a1fc-9592b4746863.png)
However I wish to let you know that, *this^^* is a thing

Reply:
Vanilla has separate key fields for all its registries, so whatever